### DISCUSSION TITLE
Add boss data for 大決戦(Eliminate)

### DISCUSSION THREAD

**Discussion by phpnumber1** [lonqie/SchaleDB#44] [Questions] [answered]
I tried to calculate the damage to Shiro&Kuro(LightArmed) on your site, but it seems that it's not supported on this site.
"大決戦" is only available on JP Server now, but it will soon become available on Global Server soon. So I suggest that we should do something to change this situation.

**Comment by lonqie** [comment, marked as answer]
There already exists the ability to override the Armor type of a target by clicking on it to open a dropdown menu.
Boss stats are otherwise exactly the same as their Total Assault versions so there is no need to add them as separate enemies.

![Screenshot 2023-09-22 151450](https://github.com/lonqie/SchaleDB/assets/95139223/c562a686-6f99-4243-988a-4c09f4c083ff)
